### PR TITLE
docs: misc changes

### PR DESCRIPTION
**Pull request by ArchaeotheriumSapienter**
Added the --version on checking if the commands are working if the shortcut -v does not work for users. (E.g git -v for me).

### CODE REVIEW

File: src/installing-from-source.rst [hunk 1]
```diff
@@ -41,12 +41,12 @@ You need:
...
-- :code:`python -V`
-- :code:`poetry -v`
-- :code:`node -v` / :code:`npm -v`
-- :code:`rustc -V` / :code:`cargo -v`
+- :code:`git -v` or :code:`git --version`
+- :code:`make -v` or :code:`make --version`
+- :code:`python -V` or :code:`python --version`
+- :code:`poetry -v` or :code:`poetry --version`
+- :code:`node -v` / :code:`npm -v` or :code:`node --version` / :code:`npm --version`
+- :code:`rustc -V` / :code:`cargo -v` or :code:`rustc --version` / :code:`cargo --version`
```
Comment (ErikBjare) [inline src/installing-from-source.rst:49]:
```suggestion
- :code:`git --version`
- :code:`make --version`
- :code:`python --version`
- :code:`poetry --version`
- :code:`node --version` / :code:`npm --version`
- :code:`rustc --version` / :code:`cargo --version`
```

No need to keep the shorthands.

**Comment by ErikBjare** [comment]
Looks good! Let me know when you're done with this batch of changes, and I'll merge it :)

**Comment by ArchaeotheriumSapienter** [comment]
sure I'm done with it now :) thank you!

**Comment by ArchaeotheriumSapienter** [comment]
I'm sorry for the closing. I got confused and thought you're supposed to close when it's okay to merge and not after merging. Lesson learned, and I'm sorry for the inconvenience. This branch is okay to merge now @ErikBjare :)

**Pull request merged**